packages/fxa-content-server/app/tests/spec/lib/channels/receivers/web-channel.js

```diff
@@ -178,22 +177,20 @@ describe('lib/channels/receivers/web-channel', () => {
 
     beforeEach(() => {
       sandbox = sinon.sandbox.create();
-      sandbox.spy(Raven, 'captureMessage');
```

Comment:
Removed instead of replacing with the `@sentry/browser` package here because we aren't able to spy on es6 modules as they are read only. I think the way to test this properly would be to spy on `/app/scripts/lib/sentry.js` once it is used in `web-channel.js`. Filed https://github.com/mozilla/fxa/issues/3419